src/components/Search/SearchResults/SearchResults.tsx

```diff
@@ -40,11 +41,14 @@ const SearchResults = ({
       <ul data-testid="search-results" className={`${px}-search__results-container`}>
         {isLoading ? <li className={`${px}-search__result`}>{loadingText}</li> : null}
         {hasResults &&
-          autoCompleteResults.map((result) => {
+          autoCompleteResults.map((result, i) => {
+            i++;
```

Comment:
why is this incrementing necessary?

Reply:
I guess it really isn't but then the testId of the first result is search-result-0 😆